xml/System.Data.SqlClient/SqlParameterCollection.xml

```diff
@@ -651,7 +668,26 @@ parameters.Add("@pname", Convert.ToInt32(0));
       <Parameters />
       <Docs>
         <summary>Removes all the <see cref="T:System.Data.SqlClient.SqlParameter" /> objects from the <see cref="T:System.Data.SqlClient.SqlParameterCollection" />.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Data.SqlClient.SqlParameterCollection> instance is cast to an <xref:System.Collections.IList> interface.
+ 
+ Implementations of this method can vary in how they handle the <xref:System.Collections.ICollection.Count%2A?displayProperty=nameWithType> and the capacity of a collection. Typically, the count is set to zero, and references to other objects from elements of the collection are also released. The capacity can be set to zero or a default value, or it can remain unchanged.   
+ 
+ 
+ ## Examples
+ 
+ The following example demonstrates the implementation of the <xref:System.Collections.IList> interface to create a simple list, fixed-size list. This code is part of a larger example for the <xref:System.Collections.IList> interface.
```

Comment:
"to create a simple list, fixed-size list." -> "... to create a simple, fixed-size list."
I see this pattern in 8 places that would need to be corrected..

Reply:
Thanks for noticing. I'll replace the instances.